components/browser/menu/src/main/java/mozilla/components/browser/menu/item/WebExtensionBrowserMenuItem.kt

```diff
@@ -54,7 +54,7 @@ class WebExtensionBrowserMenuItem(
         }
         badgeView.setBadgeText(action.badgeText)
         action.badgeTextColor?.let { badgeView.setTextColor(it) }
-        action.badgeBackgroundColor?.let { badgeView.setBackgroundColor(it) }
+        action.badgeBackgroundColor?.let { badgeView.background.setTint(it) }
```

Comment:
Let's have a test to verify that every time we have a `badgeBackgroundColor` we are setting the tint.

Reply:
I've tried to write that test but badgeView.background returns null and i can't test it. What should i do?

![image](https://user-images.githubusercontent.com/17825767/88718358-89dcc500-d12a-11ea-8523-b97225e53c65.png)

Reply:
It's `null` because in the test `badgeView` is mock and the background is not provided, you can create a real object         `val badgeView = TextView(testContext)` in the test

Reply:
I've changed val badgeView: TextView = spy(TextView(testContext)) with val badgeView = TextView(testContext) but no luck. It's null.

Reply:
I think you have to assign a value as by default is null :)

Reply:
I've assigned now but Line 195 throwing null :(
Could you please take a look, this is driving me crazy